content/docs/dvclive/api-reference/live/index.md

```diff
@@ -71,6 +72,9 @@ You can use `Live()` as a context manager. When exiting the context manager,
 
   If `report` is `None`, `Live.make_report()` won't generate anything.
 
+- `save_dvc_exp` - If `True`, DVCLive will create a DVC <abbr>experiment</abbr>
+  as part of `live.end()`.
```

Comment:
I think we are missing docs for `live.end()`

Reply:
Should probably also mention that it gets ignored when running a dvc pipeline via `exp run` or `repro`.

Reply:
If we add `live.end()` here, we can close #4095

Reply:
Link to some other place (cmd ref or guide?) where this behavior is explained? Or do so later from another PR which adds that info.